src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arraySorceString = sourceString
+    .split(';')
```

Comment:
It's a good practice to filter out any empty style declarations before mapping them. You can do this by checking if the trimmed string is not empty.